packages/babel-plugin/src/utils/css-builders.ts

```diff
@@ -688,11 +691,47 @@ function extractMemberExpression(
   fallbackToEvaluate = true
 ): CSSOutput | undefined {
   const bindingIdentifier = findBindingIdentifier(node);
-  if (bindingIdentifier && meta.state.cssMap[bindingIdentifier.name]) {
-    return {
-      css: [{ type: 'map', expression: node, name: bindingIdentifier.name, css: '' }],
-      variables: [],
-    };
+
+  if (bindingIdentifier) {
+    /**
+     * If we don't yet have a `meta.state.cssMap[…]` built yet, build it and cache it, eg. in this scenario:
+     * ```tsx
+     * const Component = () => <div css={styles.root} />
+     * const styles = cssMap({ root: { padding: 0 } });
+     * ```
+     *
+     * If we don't hit this, we put it into `ignoreMemberExpressions` to ignore on future runs.
```

Comment:
Not the most performant thing to do, but I couldn't find a better way that works with `cssMap` defined in different scopes, eg. not hoisted within Jest files as `packages/react/src/create-strict-api/__tests__/index.test.tsx` failed if I injected this into `Program` => `path.traverse()` instead.